src/libdrakvuf/libdrakvuf.h

```diff
@@ -317,6 +325,7 @@ os_t drakvuf_get_os_type(drakvuf_t drakvuf);
 page_mode_t drakvuf_get_page_mode(drakvuf_t drakvuf);
 int drakvuf_get_address_width(drakvuf_t drakvuf);
 const char* drakvuf_get_rekall_profile(drakvuf_t drakvuf);
+json_object* drakvuf_get_rekall_profile_json(drakvuf_t drakvuf);
```

Comment:
Is this new function needed? I don't see any callers and if it was not in `libdrakvuf.h` the callers wouldn't have to include the json-c header.

Reply:
I added `rekall_` prefixes for all these functions.